adapters/repos/db/shard.go

```diff
@@ -136,6 +136,8 @@ func NewShard(ctx context.Context, promMetrics *monitoring.PrometheusMetrics,
 			distProv = distancer.NewCosineDistanceProvider()
 		case hnswent.DistanceDot:
 			distProv = distancer.NewDotProductProvider()
+		case hnswent.DistanceGeo:
+			distProv = distancer.NewGeoProvider()
```

Comment:
I think that in default case in line 149 the error message should include `geo` in it, like that:
```
"choose one of [\"cosine\", \"dot\", \"l2-squared\", \"manhattan\",\"hamming\",\"geo\""
```